internal/pkg/deploy/cloudformation/project.go

```diff
@@ -439,3 +439,45 @@ func (cf CloudFormation) waitForStackSetOperation(stackSetName, operationID stri
 		time.Sleep(3 * time.Second)
 	}
 }
+
+// DeleteProject deletes all project specific StackSet and Stack resources.
+func (cf CloudFormation) DeleteProject(name string, accounts, regions []string) error {
+	if _, err := cf.client.DeleteStackInstances(&cloudformation.DeleteStackInstancesInput{
+		Accounts:     aws.StringSlice(accounts),
+		RetainStacks: aws.Bool(false),
+		Regions:      aws.StringSlice(regions),
+		StackSetName: aws.String(fmt.Sprintf("%s-infrastructure", name)),
+	}); err != nil {
+		return err
```

Comment:
nit: can we wrap this error with additional context? maybe the list of accounts and regions.